WolvenKit.RED4/Types/Pools/NodeRefPool.cs

```diff
@@ -1,14 +1,19 @@
 ﻿using System.Collections.Concurrent;
 using System.Text;
 using WolvenKit.Common.FNV1A;
+using WolvenKit.Core.Interfaces;
 
 namespace WolvenKit.RED4.Types;
 
 public static class NodeRefPool
 {
+    public static ILoggerService? Logger;
```

Comment:
is this assigned anywhere?

Reply:
No, not yet. Should have commented that out for now...